### ISSUE TITLE
extended.system HCRLateAttachWorkload_0 Java 13 Unsupported class file major version 57

### ISSUE THREAD

**Issue by pshipton** [eclipse/openj9#5775]
https://ci.eclipse.org/openj9/job/Test_openjdknext_j9_extended.system_ppc64le_linux_OpenJDK/1
https://ci.eclipse.org/openj9/job/Test_openjdknext_j9_extended.system_x86-64_linux_xl_OpenJDK/1
https://ci.eclipse.org/openj9/job/Test_openjdknext_j9_extended.system_x86-64_mac_OpenJDK/1
https://ci.eclipse.org/openj9/job/Test_openjdknext_j9_extended.system_x86-64_linux_OpenJDK/1
https://ci.eclipse.org/openj9/job/Test_openjdknext_j9_extended.system_s390x_linux_OpenJDK/1
```
LT  stderr Exception in thread "Attachment 40937" java.lang.reflect.InvocationTargetException
LT  stderr 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
LT  stderr 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
LT  stderr 	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
LT  stderr 	at java.base/java.lang.reflect.Method.invoke(Method.java:567)
LT  stderr 	at java.instrument/sun.instrument.InstrumentationImpl.loadClassAndStartAgent(InstrumentationImpl.java:513)
LT  stderr 	at java.instrument/sun.instrument.InstrumentationImpl.loadClassAndCallAgentmain(InstrumentationImpl.java:535)
LT  stderr 	at java.base/com.ibm.tools.attach.target.Attachment.loadAgentLibraryImpl(Native Method)
LT  stderr 	at java.base/com.ibm.tools.attach.target.Attachment.loadAgentLibrary(Attachment.java:365)
LT  stderr 	at java.base/com.ibm.tools.attach.target.Attachment.parseLoadAgent(Attachment.java:343)
LT  stderr 	at java.base/com.ibm.tools.attach.target.Attachment.doCommand(Attachment.java:187)
LT  stderr 	at java.base/com.ibm.tools.attach.target.Attachment.run(Attachment.java:147)
LT  stderr Caused by: java.lang.Error: java.lang.Error: java.lang.IllegalArgumentException: Unsupported class file major version 57
LT  stderr 	at net.adoptopenjdk.test.hcrAgent.agent.AgentLogger.throwErrors(AgentLogger.java:52)
LT  stderr 	at net.adoptopenjdk.test.hcrAgent.agent.StringModifierAgent.agentmain(StringModifierAgent.java:131)
LT  stderr 	... 11 more
LT  stderr Caused by: java.lang.Error: java.lang.IllegalArgumentException: Unsupported class file major version 57
LT  stderr 	at net.adoptopenjdk.test.hcrAgent.agent.StringTransformer.transform(StringTransformer.java:84)
LT  stderr 	at java.instrument/java.lang.instrument.ClassFileTransformer.transform(ClassFileTransformer.java:246)
LT  stderr 	at java.instrument/sun.instrument.TransformerManager.transform(TransformerManager.java:188)
LT  stderr 	at java.instrument/sun.instrument.InstrumentationImpl.transform(InstrumentationImpl.java:563)
LT  stderr 	at java.instrument/sun.instrument.InstrumentationImpl.retransformClasses0(Native Method)
LT  stderr 	at java.instrument/sun.instrument.InstrumentationImpl.retransformClasses(InstrumentationImpl.java:167)
LT  stderr 	at net.adoptopenjdk.test.hcrAgent.agent.TransformerMakerThread.run(TransformerMakerThread.java:68)
LT  stderr Caused by: java.lang.IllegalArgumentException: Unsupported class file major version 57
LT  stderr 	at org.objectweb.asm.ClassReader.<init>(ClassReader.java:184)
LT  stderr 	at org.objectweb.asm.ClassReader.<init>(ClassReader.java:166)
LT  stderr 	at org.objectweb.asm.ClassReader.<init>(ClassReader.java:152)
LT  stderr 	at net.adoptopenjdk.test.hcrAgent.agent.StringTransformer.transform(StringTransformer.java:70)
LT  stderr 	... 6 more
LT  stderr Agent failed to start!
```

**Comment by pshipton** [comment]
@JasonFengJ9

**Comment by JasonFengJ9** [comment]
This is a test issue. `asm-7.0.jar` used by the test doesn't support `JDK13`.
```
    if (checkClassVersion && readShort(classFileOffset + 6) > Opcodes.V12) {
      throw new IllegalArgumentException(
          "Unsupported class file major version " + readShort(classFileOffset + 6));
    }
```

**Comment by pshipton** [comment]
@Mesbah-Alam

**Comment by Mesbah-Alam** [comment]
System test prereqs should be using ASM v7.1 which likely has Java 13 bytecode support: https://asm.ow2.io/versions.html

**Comment by Mesbah-Alam** [comment]
This issue should be fixed now since we have upgraded ASM to 7.1 https://github.com/AdoptOpenJDK/openjdk-systemtest/issues/261
